system_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+
+package cagent
+
+import (
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows/registry"
+)
+
+func getProcessorModelName() (string, error) {
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `HARDWARE\DESCRIPTION\System\CentralProcessor\0`, registry.QUERY_VALUE)
```

Comment:
Please use a more descriptive name than `k`

Reply:
Although Golang coding guidelines recommend using short variables names when the scope is small, I love descriptive variable names too.   Fixed :+1:

Reply:
Short is good but not _too_ short :)